bigtable/write_samples.rb

```diff
@@ -43,7 +45,9 @@ def write_batch project_id, instance_id, table_id
   #  [START bigtable_writes_batch]
   table = bigtable.table instance_id, table_id
   column_family = "stats_summary"
-  timestamp = (Time.now.to_f * 1_000_000).round { -3 }
+  # rubocop:disable MethodCallWithArgsParentheses
```

Comment:
Do you want the "rubocop:disable" in the code example? I would expect it to be listed before the "[START bigtable_writes_batch]" and the "rubocop:enable" after the "[END bigtable_writes_batch]"